pkg/k8shandler/certificates.go

```diff
@@ -13,8 +13,7 @@ import (
   metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func extractMasterCertificate(namespace string, secretName string)(error) {
-
+func extractFromSecret(namespace string, secretName string, key string, toFile string) error {
```

Comment:
consider renaming to something like `extractSecretToFile` since we are not directly returning the value for our use